test/validation/api/ipsec/ipsec.c

```diff
@@ -642,9 +688,11 @@ static int recv_pkts_inline(const ipsec_test_part *part,
 			CU_ASSERT(odp_event_is_valid(ev) == 1);
 			CU_ASSERT_EQUAL(ODP_EVENT_PACKET,
 					odp_event_types(ev, &subtype));
-			CU_ASSERT_EQUAL(ODP_EVENT_PACKET_BASIC,
-					subtype);
-			CU_ASSERT(part->out[i].status.error.sa_lookup);
+			if (ODP_EVENT_PACKET_IPSEC != subtype) {
```

Comment:
This change looks wrong since we should never get ipsec subtype events from a pktio queue. If a packet was not ipsec-processed, it comes through the pktio queue as plain packet and if it was ipsec-processed it does not come through the pktio queue at all but through the SA queue.

This change also does not seem to be related to the topic of the PR.

Reply:
Ack. Reverted the change with v11.